src/zope/component/tests/test__declaration.py

```diff
@@ -80,22 +80,11 @@ def _run_generated_code(self, code, globs, locs,
                             fails_under_py3k=True,
                            ):
         import warnings
-        #from zope.component._compat import PYTHON3
-        PYTHON3 = False
```

Comment:
This whole testcase class should be wrapped up in a `skipIf(sys.version_info[0] > 2)` check.